jdk/make/CopyFiles.gmk

```diff
@@ -251,53 +251,77 @@ endif
 
 ##########################################################################################
 # Optionally copy OpenSSL Crypto Lbrary
+# To bundle first search for openssl 1.1.x library, if not found, search for 1.0.x 
 ifneq ($(OPENSSL_BUNDLE_LIB_PATH), )
   ifeq ($(OPENJDK_TARGET_OS), linux)
     OPENSSL_LIB_NAME = libcrypto.so.1.1
+    ifeq ("$(wildcard $(OPENSSL_BUNDLE_LIB_PATH)/$(OPENSSL_LIB_NAME))", "")
+      OPENSSL_LIB_NAME = libcrypto.so.1.0.0
+      ifeq ("$(wildcard $(OPENSSL_BUNDLE_LIB_PATH)/$(OPENSSL_LIB_NAME))", "")
+        OPENSSL_LIB_NAME = 
+      endif
+    endif
   else ifeq ($(OPENJDK_TARGET_OS), macosx)
     OPENSSL_LIB_NAME = libcrypto.1.1.dylib
+    ifeq ("$(wildcard $(OPENSSL_BUNDLE_LIB_PATH)/$(OPENSSL_LIB_NAME))", "")
+      OPENSSL_LIB_NAME = libcrypto.1.0.0.dylib
+      ifeq ("$(wildcard $(OPENSSL_BUNDLE_LIB_PATH)/$(OPENSSL_LIB_NAME))", "")
+        OPENSSL_LIB_NAME = 
+      endif
+    endif
   else ifeq ($(OPENJDK_TARGET_OS), windows)
     ifeq ($(OPENJDK_TARGET_CPU_BITS), 64)
       OPENSSL_LIB_NAME = libcrypto-1_1-x64.dll
     else
       OPENSSL_LIB_NAME = libcrypto-1_1.dll
     endif
+    ifeq ("$(wildcard $(OPENSSL_BUNDLE_LIB_PATH)/$(OPENSSL_LIB_NAME))", "")
+      OPENSSL_LIB_NAME = libeay32.dll
+      ifeq ("$(wildcard $(OPENSSL_BUNDLE_LIB_PATH)/$(OPENSSL_LIB_NAME))", "")
+        OPENSSL_LIB_NAME = 
+      endif
+    endif
   else ifeq ($(OPENJDK_TARGET_OS), aix)
     # OpenSSL 1.1.1 on AIX has switched to use archive library files (natural way)
     # instead of 'libcrypto.so' files.
     # For reference, corresponding OpenSSL PR is
     #     https://github.com/openssl/openssl/pull/6487
-    # For OpenSSL v1.1.0, bundle libcrypto.so library with JDK
-    OPENSSL_LIB_NAME = libcrypto.so
+    # For OpenSSL v1.0.0, bundle libcrypto.so library with JDK
+    OPENSSL_LIB_NAME = libcrypto.so.1.1
```

Comment:
This seems to be missing a path that selects `libcrypto.so.1.0.0` as is done on Linux.

Reply:
Fixed